app/src/main/java/org/mozilla/webmaker/javascript/WebAppInterface.java

```diff
@@ -300,6 +301,6 @@ public boolean isDebugBuild() {
      */
     @JavascriptInterface
     public String getSystemLocale() {
-        return Locale.getDefault().toString();
+        return Locale.getDefault().toString().replaceAll(String.valueOf(Pattern.compile("[_]")), "-");
```

Comment:
Since we're only replacing a delimited, I don't think we need the regexp, a straight up `...toString().replaceAll("_","-")` should do the trick